README.md

```diff
@@ -447,7 +447,7 @@ Below are the major development axis identified for Dyna&omega;o for the next fe
 
 ### Axis 3 - Dependencies, architecture and performances
 * ~~Switch to OpenModelica V1.13 version and DAE mode use~~
-* Switch to Sundials V5.0.0 and Suitesparse V5.3.0
+* ~~Switch to Sundials V5.0.0 and Suitesparse V5.3.0~~
```

Comment:
I am confused with the versioning, is it 5.0.0 or 5.3.0 for sundials ? suitesparse does not appear to change for now

Reply:
And in dynawo-docker you removed version 4.1.0 to put 5.0.0 and 5.3.0 for sundials but it breaks the ci as 5.0 was never integrated

Reply:
Weird, I was under the impression that the upgrade of suiteparse was already done. I will do it as well